tests/test_xero_bookmarks.py

```diff
@@ -72,3 +72,29 @@ def test_run(self):
         self.check_offsets(bookmarks)
         self.look_for_unexpected_bookmarks(bookmarks)
         self.assertIsNone(state.get("currently_syncing"))
+
+
+class TestArchivedContacts(XeroBookmarks):
+
+    def get_properties(self):
+        properties = super().get_properties()
+        properties["includeArchivedContacts"] = "true"
+
+        return properties
+
+
+    def test_get_archived_contacts(self):
```

Comment:
If we want to put regression protection in place for https://jira.talendforge.org/browse/TDL-592 we would need to run a sync without the `includeArchivedContacts` parameter and another sync with it set to true. Then we can confirm that the second sync results in more records replicated than the first.

Reply:
Using the test data in the stitch xero test  account, we see 10 records returned when setting the parameter to "true". When running a sync without setting the parameter we get back 8 contacts. So I think a good test would be what I described in the previous comment, with two additional  assertions. After the first sync (without the parameter) we assert all ContactStatus values are "Active". After the second sync (with the parameter) we assert the new records have ContactStatus values of "Archived". Records can be compared across syncs using their primary keys of `ContactID`. A new connection will be needed in order to run the second sync with the updated configurable property.

Reply:
@KAllan357 Thanks for your suggestion.
I have incorporated the code change as suggested in the new test file "test_archived_contacts_support.py" file.